parsec/core/fs/remote_loader.py

```diff
@@ -517,12 +519,12 @@ def __init__(self, remote_loader: RemoteLoader, timestamp: Pendulum):
         self.timestamp = timestamp
 
     async def upload_block(self, *e, **ke):
-        raise FSError(f"Cannot upload block through a timestamped remote loader")
+        raise FSError("Cannot upload block through a timestamped remote loader")
 
     async def load_manifest(
         self,
         entry_id: EntryID,
-        version: int = None,
+        version: Optional[int] = None,
         timestamp: Pendulum = None,
```

Comment:
optional on `timestamp` and `expected_backend_timestamp` too

Reply:
fixed